scripts/v.dissolve/v.dissolve.py

```diff
@@ -95,8 +398,17 @@ def main():
 
         if coltype["type"] not in ("INTEGER", "SMALLINT", "CHARACTER", "TEXT"):
             grass.fatal(_("Key column must be of type integer or string"))
+        column_is_str = bool(coltype["type"] in ("CHARACTER", "TEXT"))
```

Comment:
Need `bool()`?

Reply:
I don't think I need bool here. No idea why I put it there. Removed.